code-challenges.js

```diff
@@ -31,38 +49,82 @@ const temp2 = 350
 const temp3 = 212
 // Expected output: "212 is at boiling point"
 
+const boiling = (value) => {
+
+    if (value ===212){
+        return `${value} is at boiling point`}
+    else if (value< 212) {
+        return `${value} is below boiling point`
+    }
+    else if (value > 212){
+        return `${value} is above boiling point`
+    }
+    
+   
+}   
+    
+console.log(boiling(temp1))
+// output : 42 is below boiling point
+
+console.log(boiling(temp2))
+// output : 350 is below boiling point
+
+console.log(boiling(temp3))
+// output : 212 is at boiling point
+
+
 // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
 
-// Pseudo code:
+// Pseudo code: 
+// process: combine two arrays using concat into a variable and return the length using .length in a console log.
+
 
 const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
 const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+const padresRuns = (padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+console.log(padresRuns.length)
+// output: 9
+
 // Expected output: 9
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
 
-// Pseudo code:
+// Pseudo code: turn the string into an array and reverse using the built in mutator .reverse() and ,split("") in a single console log
+
 
 const currentCohort = "Hotel 2022"
+console.log(currentCohort.split("").reverse())
```

Comment:
This is soooo close! This console.log as is will actually return: ['2', '2', '0', '2',' ', 'l', 'e', 't','o', 'H']. You have all of your logic correct here, just missing that last step of joining (.join("")) the separated strings back together.